Return error on missing event instead of panicking. (#3944)
rust-windowing/winit: src/changelog/unreleased.md

    the primary finger in a multi-touch interaction.
  - In the same spirit rename `DeviceEvent::MouseMotion` to `PointerMotion`.
  - Remove `Force::Calibrated::altitude_angle`.

> - Change signature of `apple::appkit::WindowDelegate::drag_window()` to return `Result<(), RequestError>`.

Reviewer: Internal changes are not relevant for users. So remove this.

Author: Agreed. Removed.

- On Orbital, `MonitorHandle::name()` now returns `None` instead of a dummy name.
- On macOS, fix `WindowEvent::Moved` sometimes being triggered unnecessarily on resize.
- On macOS, package manifest definitions of `LSUIElement` will no longer be overridden with the
  default activation policy, unless explicitly provided during initialization.

> - On macOS `drag_window()` without a left click present, will result in error instead of panic.

Reviewer: ```suggestion
- On macOS, fix crash when calling `drag_window()` without a left click present.
```
